beacon/beacon.go

```diff
@@ -795,7 +800,12 @@ func (pd *ProtocolDriver) sendFirstRoundVote(ctx context.Context, epoch types.Ep
 		types.FirstRound,
 		log.String("message", m.String()))
 
-	pd.sendToGossip(ctx, pubsub.BeaconFirstVotesProtocol, m)
+	serialized, err := codec.Encode(&m)
```

Comment:
do you want to move 803-806 into pd.sendToGossip()? the code here and below are repetitive

Reply:
nope, I don't :) scale codec doesn't work with `interface{}`